commercetools-models/src/main/java/io/sphere/sdk/producttypes/ProductTypeDraft.java

```diff
@@ -14,8 +14,7 @@
  */
 @JsonDeserialize(as = ProductTypeDraftDsl.class)
 @ResourceDraftValue(factoryMethods = {
-        @FactoryMethod(parameterNames = {"key", "name", "description", "attributes"}),
-        @FactoryMethod(methodName = "ofKey", parameterNames = {"key"})
```

Comment:
At the creation of this generated factory method I thought `key` is the only mandatory parameter. Then I used it and got exceptions due to missing data for product types. I allowed myself to remove this method again since it has not been released yet.